data/xp-pen-deco-mini7.tablet

```diff
@@ -0,0 +1,18 @@
+# XP-Pen
+# Deco mini7
+#
```

Comment:
missing the sysinfo, see e.g. 9d61a6c0cf4c0052a53afa288a4bf900e4eb2018 on how to add this